apps/src/schoolInfo/utils/sendAnalyticsEvent.ts

```diff
@@ -0,0 +1,9 @@
+import {PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+
+export function sendAnalyticsEvent(
+  eventName: string,
+  data: Record<string, string>
+) {
+  analyticsReporter.sendEvent(eventName, data, PLATFORMS.BOTH);
+}
```

Comment:
Is this file necessary? It seems like it takes up the same amount of space for where it's used and adds an extra step

Reply:
The nice part of moving it to its own file is it makes it much easier to mock when testing components that use it

Reply:
In general, my vote would be to match the pattern of our other uses of sendEvent in our codebase. I think the additional layer of abstraction living in a different file makes it harder to understand the analytics at a glance. That said I respect this move if you have strong feelings about it!

Reply:
fair enough! I can remove the util function and just call the `analyticsReporter.sendEvent` method directly where it's needed